Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10 AS builder
+WORKDIR /app
+COPY requirements.txt .
+RUN python3 -m venv .venv && .venv/bin/pip install --no-cache-dir -r requirements.txt
```

Comment:
if we use docker do we still need venv？

Reply:
This is a good question. Honestly, I ported this Dockerfile from the xiaogpt project, and I didn't think about it that much originally.
After reviewing some information, I found that most people believe that there is no need to use virtual environments like venv in Docker because Docker itself already accomplishes similar tasks. I will modify Docker accordingly and remove venv.